### ISSUE TITLE
MonitorControl consumes combo keypresses (⌘ + F15) instead of only F15/F14

### ISSUE THREAD

**Issue by varenc** [MonitorControl/MonitorControl#211]
I have an obscure global keyboard shortcut setup on ⌘+F15.  MonitorControl consumes any presses for media keys like F15, even when they're used with modifiers keys that have on effect on MonitorControl.

MonitorControl should only react when media keys are pressed by themselves, or when pressed with modifiers MonitorControl recognizes.  So F15, Ctrl+Shift+F15 (small increment) and Ctrl+F15 (internal display), should all continue to work, but ⌘+F15, Ctrl+⌥+F15, etc, should not work and should not be consumed/blocked by MonitorControl.

Not sure if MediaKeyTap already supports aborting a key press consumption or not?  But if it does, then fixing this might be as simple as observing unrecognized modifier key combinations in the [delegate method here](https://github.com/the0neyouseek/MonitorControl/blob/5ed77ab05f7d794719043a7384c295fd8ef0204b/MonitorControl/AppDelegate.swift#L200), and aborting.

**Comment by fancymans** [comment]
Glad I was able to find out the culprit for killing my ⌘ + F15 custom app shortcut haha. Was wondering what it was that caused this issue. Agree it's relatively minor but would be very nice if it didn't block modifiers. Will have to map my shortcut to something else in the meantime.

**Comment by fono** [comment]
+1 for this. For me brightness control works perfectly with the dedicated brightness keys on the external keyboard (F1, F2), and even touchbar brightness controls are working great. Thus, I don't need F14 and F15 to control brightness, but I was using them for other purposes which I would still need. It would be nice to have an option to disable these 'secondary' hotkeys, if it's not feasible to make it fully customizable. Regardless, this is an excellent app, thank you.

**Comment by waydabber** [comment]
I think there should be an option to customize keyboard shortcuts. I am trying to focus and consolidate all issues like this into #118. I'll close this one as a duplicate now. Thank you!
